src/tests/modules/imap/imap_opt_tls/module.conf

```diff
@@ -0,0 +1,61 @@
+#IMAP unit test config
+
+# testdir      = $ENV{TESTDIR}
+# output       = $ENV{OUTPUT}
+# run_dir      = ${output}
+# raddb        = raddb
+# pidfile      = ${run_dir}/radiusd.pid
+# panic_action = "gdb -batch -x src/tests/panic.gdb %e %p > ${run_dir}/gdb.log 2>&1; cat ${run_dir}/gdb.log"
+
+# maindir      = ${raddb}
+# radacctdir   = ${run_dir}/radacct
+# modconfdir   = ${maindir}/mods-config
+# certdir      = 
+# cadir        = /certs
+module_dir	= $ENV{MODULE_TEST_DIR}
+# test_port    = $ENV{TEST_PORT}
+
+imap imap_try_tls {
+	imap_uri = "127.0.0.1:1431"
+	timeout = 1000
+
+	tls {
+                #
+                #  .Certificate validation options
+                #
+                #  Specifies how the certificate(s) presented by the HTTPS server being contacted
+                #  are validated, and which certificates (if any) to send to the HTTPS server.
+
+                #
+                #  ca_file:: PEM formatted file containing the chain to validate the HTTPS server's cert
+                #
+                #  Should usually contain a concatenation of one or more intermediary CA
+                #  files, shallowest (i.e. the one that signed the HTTPS server's cert) first, and
+                #  deepest (the root CA) last.
+                #
+                #  Providing a complete certificate chain here is the most common way of validating
+                #  the certificate presented by a HTTPS server.
+                #
+             	ca_file              = "$ENV{MODULE_TEST_DIR}../../../../../raddb/certs/rsa/ca.pem"
```

Comment:
fix random whitespace

Reply:
There must be a better way of getting the repo src dir? That's pretty horrific.

Reply:
It's definitely a bit horrible... the 'better' way would probably to create a soft link inside src/tests/modules to the certs folder, but I'm not sure where I would put the code for that link, I messaged about it yesterday

Reply:
Get @alandekok to expose BASEDIR or something and use that instead, 5 levels of backtracking is pretty fragile.

Reply:
`$ENV{top_srcdir}/raddb/.....` might work, too.